tests/test_streams.py

```diff
@@ -1135,7 +1135,7 @@ def test_at_eof(self, buffer: Any) -> None:
 
     def test_feed_data(self, buffer: Any) -> None:
         item = object()
-        buffer.feed_data(item, 1)
+        buffer.feed_data(item)
         assert [(item, 1)] == list(buffer._buffer)
```

Comment:
Looks like this becomes `0`, as the default behaviour appears to skip parsing (which is why the `object()` doesn't trigger any errors).

@webknjaz Do you think there's a chance this might break anything.

Reply:
This is again because of the DataQueue. If that is updated, then this test should be fine as well.